torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -805,14 +805,25 @@ void AliasDb::analyzeImpl(Node* node) {
       "AliasAnalysisKind::CONSERVATIVE/PURE_FUNCTION/INTERNAL_SPECIAL_CASE should already have been handled above");
   const auto& schema = node->schema();
   torch::utils::SchemaInfo schema_info(schema);
-  if (!isFrozen_ && node->hasNamedInput("training")) {
-    auto value = constant_as<bool>(node->namedInput("training"));
-    schema_info.addArgumentValue("training", !value.has_value() || *value);
-  } else if (!isFrozen_ && node->hasNamedInput("use_input_stats")) {
-    auto value = constant_as<bool>(node->namedInput("use_input_stats"));
-    schema_info.addArgumentValue(
-        "use_input_stats", !value.has_value() || *value);
+
+  // Add arguments for batch_norm and instance_norm special case.
+  if (node->hasNamedInput("training")) {
+    if (isFrozen_) {
+      schema_info.addArgumentValue("training", false);
+    } else {
+      auto value = constant_as<bool>(node->namedInput("training"));
+      schema_info.addArgumentValue("training", !value.has_value() || *value);
```

Comment:
nit: in the case that we can't find the constant value (i.e. `!value.has_value()`) and `isFrozen_ == false` can we not assign a value to schema_info? Practically it's probably not an issue here, but hypothetically you might want to change the default behavior and in that case we don't want to add the argument value here.